Source/WebKit/WebProcess/GPU/webrtc/SharedVideoFrame.h

```diff
@@ -55,7 +58,7 @@ class RemoteVideoFrameObjectHeap;
 struct SharedVideoFrame {
     MediaTime time;
     bool mirrored { false };
-    WebCore::VideoFrame::Rotation rotation { WebCore::VideoFrame::Rotation::None };
+    WebCore::VideoFrameRotation rotation { };
```

Comment:
I guess that won't work here. :/

Reply:
Indeed it won't. We're relying on zero being a valid Enum value here.

Reply:
If we’re really worried about depending on this, we could put a static assertion that it’s zero somewhere with a comment mentioning this header depends on that.